src/main/java/no/nav/familie/integrasjoner/organisasjon/OrganisasjonController.kt

```diff
@@ -22,4 +22,9 @@ class OrganisasjonController(private val organisasjonService: OrganisasjonServic
         return success(data = organisasjonService.hentOrganisasjon(orgnr))
     }
 
+    @GetMapping(path = ["/valider/{orgnr}"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Hvordan kommer denne brukes? 
Hvis bruket er att dere alltid kommer kalle valider før dere henter organisasjon, burde man kanskje vurdere en Request Param til henting av org nummer i stedet? 
Eks att man sender med `?valider=true`, sånn att den validerer først når man henter orgnummer?

Reply:
Burde det være orgnr først, for å være mer lik den andre endepunkten? 
```suggestion
    @GetMapping(path = ["{orgnr}/valider"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Reply:
Denne tjenesten legges til for å validere organisasjon nummer som sendes av frontend. Vi trenger ikke å hente organisasjon etterpå.

Reply:
Endret endepunkten til {orgnr}/valider